### ISSUE TITLE
undefined method `disable' for #<PaperTraiundefined method `disable' for #<PaperTrail::ModelConfig:0x0055f09c6670a0> (NoMethodError)l::ModelConfig:0x0055f09c6670a0> (NoMethodError)

### ISSUE THREAD

**Issue by fjorba** [rism-ch/muscat#781]
After installing 5.1, I'm trying to load some sample records using this syntax:

$ rails runner housekeeping/import/import_from_marc.rb sample-records.marcxml Source

And I'm getting this error:

```
from [...]/muscat-5.1.2/vendor/bundle/ruby/2.3.0/gems/railties-5.1.6/lib/rails/commands/runner/runner_command.rb:34:in `load'
 from [...]/muscat-5.1.2/vendor/bundle/ruby/2.3.0/gems/railties-5.1.6/lib/rails/commands/runner/runner_command.rb:34:in `perform'
 from [...]/muscat-5.1.2/vendor/bundle/ruby/2.3.0/gems/thor-0.20.0/lib/thor/command.rb:27:in `run'
 from [...]/muscat-5.1.2/vendor/bundle/ruby/2.3.0/gems/thor-0.20.0/lib/thor/invocation.rb:126:in `invoke_command'
 from [...]/muscat-5.1.2/vendor/bundle/ruby/2.3.0/gems/thor-0.20.0/lib/thor.rb:387:in `dispatch'
 from [...]/muscat-5.1.2/vendor/bundle/ruby/2.3.0/gems/railties-5.1.6/lib/rails/command/base.rb:63:in `perform'
 from [...]/muscat-5.1.2/vendor/bundle/ruby/2.3.0/gems/railties-5.1.6/lib/rails/command.rb:44:in `invoke'
 from [...]/muscat-5.1.2/vendor/bundle/ruby/2.3.0/gems/railties-5.1.6/lib/rails/commands.rb:16:in `<top (required)>'
```

This is run under Debian 9 (Stretch), except for MariaDB, running under Debian Buster (10) due to an known incompatibility (
https://www.google.com/search?q=mysql+row+size+too+large)

**Comment by fjorba** [comment]
Apparently, commenting the paper_trail lines out the records can be loaded

```
--- a/housekeeping/import/import_from_marc.rb
+++ b/housekeeping/import/import_from_marc.rb
@@ -2,11 +2,11 @@
 # second is the model name
 # third is the offset to start from

-Catalogue.paper_trail.disable
-Holding.paper_trail.disable
-Institution.paper_trail.disable
-Person.paper_trail.disable
-Source.paper_trail.disable
+# Catalogue.paper_trail.disable
+# Holding.paper_trail.disable
+# Institution.paper_trail.disable
+# Person.paper_trail.disable
+# Source.paper_trail.disable
 if ARGV.length >= 2
   source_file = ARGV[0]
   model = ARGV[1]
```

How important is this error, and is it safe to continue (our tests) with those lines commented out?

Thanks

**Comment by xhero** [comment]
Yes! you can safely disable them, it will prevent PaperTrail to make an old version of the object, which would not happen anyways as it is newly imported.
I will push a fix in 5.2 for this.

**Comment by fjorba** [comment]
Thanks again!
